lib/checkclass.cpp

```diff
@@ -272,8 +272,10 @@ void CheckClass::checkExplicitConstructors()
             if (!func.isConstructor() || func.isDelete() || (!func.hasBody() && func.access == Private))
                 continue;
 
+            const std::size_t nonDefaultArgumentsCount = func.argCount() - countDefaultArguments(func.argumentList);
+            
             if (!func.isExplicit() &&
-                func.argCount() == 1 &&
+                nonDefaultArgumentsCount == 1 &&
```

Comment:
I am not sure but doesn't the `Function::minArgCount()` return the non default arguments count?

Reply:
Oh wow, totally missed that!